spec/models/group_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+RSpec.describe Group, type: :model do
+  it { should validate_presence_of(:name) }
```

Comment:
According to http://www.betterspecs.org On one line expectations or with implicit subject we should use is_expected.to